lib/beaker/host/aix/file.rb

```diff
@@ -1,8 +1,8 @@
 module Aix::File
   include Beaker::CommandFactory
 
-  def tmpfile(name = '')
-    execute("rndnum=${RANDOM} && touch /tmp/#{name}.${rndnum} && echo /tmp/#{name}.${rndnum}")
+  def tmpfile(name = '', extension = nil)
+    execute("rndnum=${RANDOM} && touch /tmp/#{name}.${rndnum} && echo /tmp/#{name}.${rndnum}#{extension}")
```

Comment:
This looks bad (regarding the *uniqueness* of the generated temporary file name) but I guess AIX is *that* bad :shrug: 
```suggestion
    execute("rndnum=${RANDOM} && touch /tmp/#{name}.${rndnum}#{extension} && echo /tmp/#{name}.${rndnum}#{extension}")
```